<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>1018</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QGroupBox" name="options">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTabWidget" name="Settings">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="crane">
              <attribute name="title">
               <string>Crane</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QGroupBox" name="towerBox">
                 <property name="title">
                  <string>Tower</string>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="towerHeight_label">
                    <property name="text">
                     <string>Height</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="towerHeight_spinbox">
                    <property name="suffix">
                     <string> mm</string>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000000</number>
                    </property>
                    <property name="value">
                     <number>2000</number>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="towerWidth_label">
                    <property name="text">
                     <string>Width</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QSpinBox" name="towerWidth_spinbox">
                    <property name="suffix">
                     <string> mm</string>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000000</number>
                    </property>
                    <property name="value">
                     <number>1000</number>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0">
                   <widget class="QLabel" name="towerSegment_label">
                    <property name="text">
                     <string>Segments</string>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="0">
                   <widget class="QLabel" name="towerSupportType_label">
                    <property name="text">
                     <string>Support Type</string>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="1">
                   <widget class="QComboBox" name="towerSupportType_comboBox">
                    <property name="autoFillBackground">
                     <bool>false</bool>
                    </property>
                    <property name="currentText">
                     <string>Zigzag</string>
                    </property>
                    <item>
                     <property name="text">
                      <string>Zigzag</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>None</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Cross</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Diagonal</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="6" column="1">
                   <widget class="QSpinBox" name="towerSegment_spinbox">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000000</number>
                    </property>
                    <property name="value">
                     <number>2</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="jibBox">
                 <property name="title">
                  <string>Jib</string>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <layout class="QFormLayout" name="formLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="jibLength_label">
                    <property name="text">
                     <string>Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="jibLength_spinBox">
                    <property name="suffix">
                     <string> mm</string>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000000</number>
                    </property>
                    <property name="value">
                     <number>2000</number>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="jibHeight_label">
                    <property name="text">
                     <string>Height</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QSpinBox" name="jibHeight_spinBox">
                    <property name="suffix">
                     <string> mm</string>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000000</number>
                    </property>
                    <property name="value">
                     <number>1000</number>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0">
                   <widget class="QLabel" name="jibSegment_label">
                    <property name="text">
                     <string>Segments</string>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="1">
                   <widget class="QSpinBox" name="jibSegment_spinBox">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000000</number>
                    </property>
                    <property name="value">
                     <number>2</number>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="0">
                   <widget class="QLabel" name="jibSupportType_label">
                    <property name="text">
                     <string>Support Type</string>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="1">
                   <widget class="QComboBox" name="jibSupportType_comboBox">
                    <property name="currentText">
                     <string>Cross</string>
                    </property>
                    <item>
                     <property name="text">
                      <string>Cross</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>ZigZag</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Diagonal</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="counterJibBox">
                 <property name="title">
                  <string>Counter Jib</string>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <layout class="QFormLayout" name="formLayout_3">
                  <item row="0" column="0">
                   <widget class="QLabel" name="counterJibLength_label">
                    <property name="text">
                     <string>Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="counterJibLength_spinBox">
                    <property name="suffix">
                     <string> mm</string>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000000</number>
                    </property>
                    <property name="value">
                     <number>1000</number>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="counterJibHeight_label">
                    <property name="text">
                     <string>Height</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QSpinBox" name="counterJibHeight_spinBox">
                    <property name="suffix">
                     <string> mm</string>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000000</number>
                    </property>
                    <property name="value">
                     <number>800</number>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="0">
                   <widget class="QLabel" name="counterJibSupportType_label">
                    <property name="text">
                     <string>Support Type</string>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0">
                   <widget class="QLabel" name="counterJibSegments_label">
                    <property name="text">
                     <string>Segments</string>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="1">
                   <widget class="QSpinBox" name="counterJibSegments_spinBox">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000000</number>
                    </property>
                    <property name="value">
                     <number>2</number>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="1">
                   <widget class="QComboBox" name="counterJibSupportType_comboBox">
                    <property name="currentText">
                     <string>Truss</string>
                    </property>
                    <item>
                     <property name="text">
                      <string>Truss</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>None</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>One tower</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Two towers</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="settings">
              <attribute name="title">
               <string>Settings</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QCheckBox" name="enable_gravity">
                 <property name="text">
                  <string>Gravity</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="enable_wind">
                 <property name="text">
                  <string>Wind</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="enableFEM_checkbox">
                 <property name="text">
                  <string>FEM</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout_5">
                 <item row="0" column="0">
                  <widget class="QLabel" name="scale_label">
                   <property name="text">
                    <string>Scale</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSpinBox" name="scaleSpinBox">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="ignore_specification">
                 <property name="text">
                  <string>Ignore specifications of the project task</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="fem">
              <attribute name="title">
               <string>FEM</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QTreeWidget" name="fem_treeWidget">
                 <column>
                  <property name="text">
                   <string notr="true">1</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="debug">
              <attribute name="title">
               <string>Debug</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QTreeWidget" name="debug_treeWidget">
                 <column>
                  <property name="text">
                   <string notr="true">1</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="runBox">
          <property name="title">
           <string>Run</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="apply_button">
             <property name="text">
              <string>Apply</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QSplitter" name="splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGroupBox" name="plotBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Plot</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="plot_layout"/>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="outputBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>7</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPlainTextEdit" name="output">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="clear_button">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>jibLength_spinBox</tabstop>
  <tabstop>jibHeight_spinBox</tabstop>
  <tabstop>counterJibLength_spinBox</tabstop>
  <tabstop>counterJibHeight_spinBox</tabstop>
  <tabstop>output</tabstop>
  <tabstop>clear_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>output</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>906</x>
     <y>723</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>674</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
